salt/environment_settings.yml

```diff
@@ -138,6 +138,31 @@ sandbox_mitxpro_versions: &sandbox_mitxpro_versions
   codename: ironwood
   ami_id: ami-80861296
 
+rds_postgres_parameter_group_defaults: &rds_postgres_parameter_group_defaults
```

Comment:
I think it's better for these defaults to live in the orchestrate state as a Jinja mapping and then if there are additional custom parameters for a given instance then we can specify them at that point and merge them into the mapping as part of the orchestrate state. Having them defined here in YAML I think adds extra bloat to this file and creates unnecessary visual noise.

Having it as a mapping also simplifies enforcing that if there are version specific overrides then we can define them there and more atomically ensure that they get applied rather than manually tracking version updates for the engine and the parameter group in this file.

Reply:
I saw the yaml merge technique being used in `pillar/devstack.sls` and was emulating how it was done there. Can you point me at an example of how to do it with the jinja map method?

Reply:
In Jinja it's essentially just a Python dict so you would use the `.update` method to merge the dicts.

e.g.
```
{% set parameter_map = {'postgres_default_parameters': {'character_set_encoding': 'utf8'},
'mysql_default_parameters': {'character_set_encoding': 'utf8'}} %}
{% set extra_params = db_config.pop('parameters', {}) %}
{% do paramater_map.update(extra_params) %}
```

Reply:
OK, thanks 👍

Reply:
I've changed this in commits ff310fb and 7b39348.